### ISSUE TITLE
Please make it compatible with #scrollable-tabstrip

### ISSUE THREAD

**Issue by amymor** [ArnaudLigny/Open-New-Tab-After-Current-Tab#38]
hello
i enjoyed your extension but at least one problem remains.
if i enable chrome: // flags / # scrollable-tabstrip, the tabs pane will be scrolled to the last.

it also happens for settings and history tabs and etc. but opening the link in the new tab doesnt have this problem. i think because Chrome opens these tabs to the last so the browser automatically scrolls to last. can you fix it?
thanks

**Comment by ArnaudLigny** [comment]
Hello @amymor, after some tests it appears this experimental feature auto scroll to the end of the tab bar regardless of the location of the newly opened tab.

I think this is not a normal / ideal behavior, but my extension has nothing to do with it: I have no control over the tabs scroll bar. So I will try to create a issue for the Chrome dev team.

**Comment by Novadestin** [comment]
I came here for a similar reason but have found a little bit of a work around via another extension called [New Adjacent Tab](https://chrome.google.com/webstore/detail/new-adjacent-tab/fommpldicogfgegfpcoibmhkbbdpodjg). Pinning the extension to the bar allows you to left click and open a new tab without it scrolling, so maybe you could ask them about how they have things set up? Personally, I'd rather not have another thing I need to pin to the already cluttered bar lol

**Comment by amymor** [comment]
> I came here for a similar reason but have found a little bit of a work around via another extension called [New Adjacent Tab](https://chrome.google.com/webstore/detail/new-adjacent-tab/fommpldicogfgegfpcoibmhkbbdpodjg). Pinning the extension to the bar allows you to left click and open a new tab without it scrolling, so maybe you could ask them about how they have things set up? Personally, I'd rather not have another thing I need to pin to the already cluttered bar lol

yeah i use [https://chrome.google.com/webstore/detail/new-adjacent-tab/fommpldicogfgegfpcoibmhkbbdpodjg](url)

The code is open source at https://github.com/gilmoreorless/chrome-new-tab-here

**Comment by ArnaudLigny** [comment]
Hello @amymor, it should be fixed on Chrome and Edge with the last release ([0.4.6](https://github.com/ArnaudLigny/Open-New-Tab-After-Current-Tab/releases/tag/0.4.6)).

**Comment by amymor** [comment]
> Hello @amymor, it should be fixed on Chrome and Edge with the last release ([0.4.6](https://github.com/ArnaudLigny/Open-New-Tab-After-Current-Tab/releases/tag/0.4.6)).
doesnt work on ungoogled chromium 117.0.5938.89 and 115.0.5790.131, tried on fresh profile and also with new exprimental Chrome refresh 2023 design. idk, something may have been striped by ungoogled that your extension needs to work. anyway thanks for taking your time.

**Comment by ArnaudLigny** [comment]
Issue added to "Known issues" section of the readme: https://github.com/ArnaudLigny/Open-New-Tab-After-Current-Tab/edit/master/README.md#known-issues
